### PR TITLE
refine reademe

### PR DESCRIPTION
**Pull request by kevinintel**
## Type of Change
 
documentation 

## Description

refine llm runtime readme

## Expected Behavior & Potential Risk

n/a

## How has this PR been tested?

n/a

## Dependency Change?

n/a

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]

**Review by zhenwei-intel — APPROVED** [review]
LGTM

File: intel_extension_for_transformers/llm/runtime/graph/README.md [hunk 1]
```diff
@@ -1,66 +1,64 @@
...
-You can use `git clone` command like (for example, gpt-j-6b): `git clone https://huggingface.co/EleutherAI/gpt-j-6b`.
-
-Convert process had two steps: 1. get fp32 model with llama.cpp format 2. quantize the fp32 model into model with low precision (int8, int4, etc.) We recommend you to use int4 model for better LLM inference latency.
 
 ```bash
+# download fp32 model (e.g., GPT-J-6B) from Hugging Face
+git clone https://huggingface.co/EleutherAI/gpt-j-6b
+
 # convert the pytorch model to ggml format
 python scripts/convert_model.py --outtype f32 --outfile ne-f32.bin model_path/model_id
```
Comment (zhenwei-intel) [inline intel_extension_for_transformers/llm/runtime/graph/README.md:53]: we can also support `python scripts/convert_model.py --outtype f32 --outfile ne-f32.bin EleutherAI/gpt-j-6b` without downloading models.

Comment (hshen14) [reply intel_extension_for_transformers/llm/runtime/graph/README.md:53]: Good point. Let me refine it.

File: intel_extension_for_transformers/llm/runtime/graph/scripts/run_llm.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    if not path.exists():
+        print("Please build graph first or select the correct model name.")
+        sys.exit(1)
+
+    cmd = [path]
+    cmd.extend(["--model",          args.model])
+    cmd.extend(["--prompt",         args.prompt])
+    cmd.extend(["--n-predict",      str(args.n_predict)])
+    cmd.extend(["--threads",        str(args.threads)])
+    cmd.extend(["--batch-size",     str(args.batch_size)])
```
Comment (intellinjun) [inline intel_extension_for_transformers/llm/runtime/graph/scripts/run_llm.py:102]:
```suggestion
    cmd.extend(["--batch-size-truncate",     str(args.batch-size-truncate)])
```

Comment (zhenwei-intel) [reply intel_extension_for_transformers/llm/runtime/graph/scripts/run_llm.py:102]: update in this PR: https://github.com/intel/intel-extension-for-transformers/pull/320

**Pull request merged**